bookops_marc/bib.py

```diff
@@ -476,52 +377,117 @@ def subjects_lc(self) -> List[Field]:
                 lc_subjects.append(field)
         return lc_subjects
 
+    @property
     def suppressed(self) -> bool:
         """
         Determines based on 998 $e value if bib is suppressed from public display
         BPL usage: "c", "n"
         NYPL usage: "c", "e", "n", "q", "o", "v"
         """
-        try:
-            code = self.get("998").get(code="e")  # type: ignore
-        except (TypeError, AttributeError):
+        field = self.get("998")
+        if not field:
             return False
-
-        if code in ("c", "e", "n", "q", "o", "v"):
+        code = field.get("e")
+        if isinstance(code, str) and code in ("c", "e", "n", "q", "o", "v"):
             return True
         else:
             return False
 
+    @property
     def upc_number(self) -> Optional[str]:
         """
         Returns a UPC number if present on the bib.
         https://www.loc.gov/marc/bibliographic/bd024.html
         """
         tag = self.get("024")
-        if tag:
-            if tag.indicator1 == "1":
-                try:
-                    return tag.get(code="a").strip()  # type: ignore
-                except AttributeError:
-                    pass
+        if tag and tag.indicator1 and tag.indicator1 == "1":
+            upc_num = tag.get(code="a")
+            if isinstance(upc_num, str):
+                return upc_num.strip()
         return None
 
+    def sort_orders(self, sort: str = "descending") -> Sequence[Order]:
+        """
+        Returns a sorted list of orders attached to a bib.
+        Args:
```

Comment:
Let's add an explanation to this docstring that the order of 960 tags in exported MARC is from the oldest to the newest order, i.e. the first 960 tag describes the oldest, the last the newest order. The connection may not be apparent to non-Sierra-cataloger.